app/lib/actions/candlepin/candlepin_listening_service.rb

```diff
@@ -59,7 +59,7 @@ def start(suspended_action)
         else
           suspended_action.notify_not_connected("Not Connected")
         end
-      rescue TransportFailure => e
+      rescue => e
```

Comment:
@jlsherrill I don't any error when I changed to this, I think TransportFailure was being evaluated at some point and throwing uninitialized constant errors. Let me know what you think.